lte/gateway/c/oai/tasks/amf/amf_security_mode_control.cpp

```diff
@@ -221,36 +225,37 @@ static int security_mode_t3560_handler(zloop_t* loop, int timer_id, void* arg) {
         LOG_AMF_APP, "T3560 timer   timer id %d ue id %d\n", smc_proc->T3560.id,
         smc_proc->ue_id);
     smc_proc->T3560.id = -1;
-  }
-
-  /*
-   * Increment the retransmission counter
-   */
-  smc_proc->retransmission_count += 1;
-  OAILOG_ERROR(
-      LOG_AMF_APP, "Timer: Incrementing retransmission_count to %d\n",
-      smc_proc->retransmission_count);
-
-  if (smc_proc->retransmission_count < SECURITY_COUNTER_MAX) {
     /*
-     * Send identity request message to the UE
+     * Increment the retransmission counter
      */
+    smc_proc->retransmission_count += 1;
     OAILOG_ERROR(
-        LOG_AMF_APP,
-        "Timer: timer has expired Sending Security Command Mode request "
-        "again\n");
-    amf_security_request(smc_proc);
+        LOG_AMF_APP, "Timer: Incrementing retransmission_count to %d\n",
+        smc_proc->retransmission_count);
+
+    if (smc_proc->retransmission_count < SECURITY_COUNTER_MAX) {
+      /*
+       * Send identity request message to the UE
+       */
+      OAILOG_ERROR(
+          LOG_AMF_APP,
+          "Timer: timer has expired Sending Security Command Mode request "
+          "again\n");
+      amf_security_request(smc_proc);
```

Comment:
a better name can be given to the function. is it Security Mode Command? instead of security_request can this be renamed to smc or security_mode_cmd? it is not request, it is a command.